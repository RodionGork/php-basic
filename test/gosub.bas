print "before sub1"
gosub sub
print "after sub1"

print "before sub2" : gosub sub : print "after sub2"
end

sub: print "in the sub"
return
