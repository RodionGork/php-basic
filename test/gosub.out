before sub1
in the sub
after sub1
before sub2
in the sub
after sub2
